bigflow/log.py

```diff
@@ -93,12 +99,54 @@ class LogConfigDict(TypedDict):
     log_level: typing.Union[str, int]
 
 
-def _generate_cl_log_view_link(params: dict):
-    query = quote_plus("\n".join(
-        '{}="{}"'.format(k, v)
+def _generate_cl_log_view_query(params: dict, separator: str = '='):
+    query = "".join(
```

Comment:
nit: imo it is better to use `"\n".join(...)`

Reply:
Why?

Reply:
IMO it is a bit easier to read code as:
   "here we are joining set of strings with "\n" as delimiter"
instead of 
   "here we are joining set of strings, and each string ends with a "\n", so effectively it is like joining with "\n" as delimiter"

But it is more stylish preference, so feel free to keep your varian if you prefer.